build.sbt

```diff
@@ -15,8 +15,9 @@ crossScalaVersions := Seq("2.12.11", "2.11.12")
 libraryDependencies := Seq(
   "junit" % "junit" % "4.12" % "test",
   "com.novocode" % "junit-interface" % "0.11" % "test",
-  "org.apache.daffodil" %% "daffodil-tdml-lib" % "2.6.0",
-  "org.scala-lang.modules" %% "scala-xml" % "1.1.0"
+  "org.apache.daffodil" %% "daffodil-tdml-lib" % "3.1.0-SNAPSHOT",
```

Comment:
I suggest we always keep this at 3.0.0 so that non daffodil devs can easily use this. Any one that's building custom daffodil snapshots will likely know how to update this file to use those snapshots.

Reply:
While you were changing the dependencies, I hope you took the opportunity to check for newer versions and upgrade dependencies to newer versions where possible?

Reply:
Will update dependencies to match those in Daffodil 3.0.0 as those were recently updated.

This module really does depend on those changes I put into 3.1.0-SNAPSHOT. It has a unit test that fails now without that namespace support for the root element in negative parser test cases. 

It will work fine with 3.0.0, (excepting negative parser test cases), but won't get a clean 'sbt test'.

Realistically, we're the only people using this. So I think it's ok to have it depend on a snapshot.... but only after I'm able to push that commit to daffodil.